### ISSUE TITLE
Introduce protractor test

### ISSUE THREAD

**Issue by tschaka1904** [Complex-Portal/complex-portal-view#69]

**Comment by bmeldal** [comment]
???
@noedelta @elisabetb

**Comment by noedelta** [comment]
Protractor is an end-to-end test framework for Angular and AngularJS applications. I guess it was the framework that Max wanted to use for testing. We can have a look.

Noe

**Comment by tschaka1904** [comment]
Thats right! Maybe not chosen by me, but it is the recommend and default E2E testing tool. I can also recommend Jest for just some unit testing. I think, if you create a new component it will automatically generate a Protractor test file.

**Comment by tschaka1904** [comment]
I would recommend closing this. I would have been nice to have this, but it is not essential.
